### PR TITLE
Fix LPCL in SC

### PR DESCRIPTION
**Pull request by SamerKhshiboun**
Tracked in [RSDEV-280]

### CODE REVIEW

**Comment by Nir-Az** [comment]
@SamerKhshiboun will this fix the validation issue?
Ready for merge? or you want to give this fix to val to verify?

**Comment by SamerKhshiboun** [comment]
@Nir-Az I'm waiting for Yehuda response on the Jira ticket, as I prefer he will test this before we merge it.
Tracked by RSDEV-280

**Review by Nir-Az — APPROVED** [review]

**Pull request merged**
